force-app/main/default/classes/CRLP_ResetRollupFieldsQueueable.cls

```diff
@@ -0,0 +1,134 @@
+     */
+    private List<Id> recordIds;
+
+    /** @description The Summary Object Type this batch job will process
+     * Currently only supported for GAU summaryObjectType.
+     */
+    private SObjectType summaryObjectType;
+
+    public CRLP_ResetRollupFieldsQueueable(CRLP_RollupProcessingOptions.RollupType jobType, List<Id> recordIds) {
+        this.jobType = jobType;
+        this.summaryObjectType = CRLP_Rollup_SVC.getSummaryObjectType(jobType);
+        this.recordIds = recordIds;
+    }
+
+    /**
+     * @description Call clearRollups to zero out the summary record Ids passed to the job.
+     * @param qc
+     */
+    public void execute(QueueableContext qc) {
+        try {
+            clearRollupFieldValues();
+        } catch (Exception ex) {
+            ERR_Handler.processError(ex, ERR_Handler_API.Context.CRLP + '.' + this.jobType.name());
+        }
+    }
+
+    /**
+     * @description Clear summary records for records whose associated details have been deleted after a skew mode run.
+     */
+    @TestVisible
+    private void clearRollupFieldValues() {
+        List<SObject> updatedRecords = getUpdatedRecords(getSummaryRecords());
+        if (!updatedRecords.isEmpty()) {
+            CRLP_RollupProcessor_SVC.updateChangedRecordsWithLogging(
+                clearSkewModeFlag(updatedRecords),
+                jobType
+            );
+        }
+    }
+
+    /**
+     * @description Get updated summary records.
+     * @return List<SObject>
+     */
+    @TestVisible
+    private List<SObject> getUpdatedRecords(List<SObject> parents) {
+        CRLP_RollupProcessingOptions.ProcessingOptions options = new CRLP_RollupProcessingOptions.ProcessingOptions();
+        options.useRollupDefsByParentIdMap = true;
+
+        CRLP_RollupProcessor processor = new CRLP_RollupProcessor()
+            .withRollupType(jobType)
+            .withDetailRecords(new List<SObject>())
+            .withRelatedRecords(new Map<Id, List<SObject>>())
+            .withJobOptions(options);
+
+        List<SObject> updatedRecords = new List<SObject>();
+        for (SObject parent : parents) {
+            SObject updatedRecord = processor.withSummaryRecord(parent).completeRollupForSingleSummaryRecord(parent);
+            if (updatedRecord != null) {
+                updatedRecords.add(updatedRecord);
+            }
+        }
+        return updatedRecords;
+    }
+
+    /**
+     * @description Query summary records by Id.
+     * @return List<SObject>
+     */
+    private List<SObject> getSummaryRecords() {
+        String query = CRLP_Query_SEL.buildObjectQueryForRollup(summaryObjectType) + ' WHERE Id IN :recordIds';
+        List<SObject> parents = Database.query(query);
+        return parents;
+    }
+
+    /**
+     * @description Unconditionally set skew mode flag false.
+     * @return List<SObject>
+     */
+    @TestVisible
+    private List<SObject> clearSkewModeFlag(List<SObject> updatedRecords) {
```

Comment:
Since it's new functionality, can we add a small Unit test for this? Or, since we're only calling this on GAUs now, we could just remove it...